db/comdb2.h

```diff
@@ -1225,6 +1211,89 @@ typedef struct bpfunc_lstnode {
 
 typedef LISTC_T(bpfunc_lstnode_t) bpfunc_list_t;
 /*******************************************************************/
+
+enum OSQL_REQ_TYPE {
+    OSQL_REQINV = 0,
+    OSQL_BLOCK_REQ = 1, /* obsolete */
+    OSQL_SOCK_REQ = 2,
+    OSQL_RECOM_REQ = 3,
+    OSQL_SERIAL_REQ = 4,
+
+    OSQL_BLOCK_REQ_COST = 5, /* obsolete */
+    OSQL_SOCK_REQ_COST = 6,
+
+    OSQL_SNAPISOL_REQ = 7,
+    OSQL_SNAP_UID_REQ = 8,
+    OSQL_MAX_REQ = 9,
+};
+
+/* Magic rqid value that means "please use uuid instead" */
+#define OSQL_RQID_USE_UUID 1
+
+struct osql_sess {
```

Comment:
this will be split in two, hiding fields that do not need exposure to rest of the world